src/main/java/com/google/sps/servlets/MessageServlet.java

```diff
@@ -83,52 +91,4 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
-        String jsonString = IOUtils.toString(request.getInputStream());
-        Message newMessage = new Gson().fromJson(jsonString, Message.class);
-        String chatroomID = newMessage.chatroomId;
-
-        // If no chatroom exists, create one.
-        if (chatroomID.equals("")) {
-            chatroomID = UUID.randomUUID().toString();
-            Chatroom newChatroom =
-                    new Chatroom(chatroomID, newMessage.senderId, newMessage.recipientId);
-            newChatroom.setEntity();
-        }
-
-        // Query the user and retrieve the language of the recipient.
-        Query userQuery = new Query("user");
-        userQuery.setFilter(
-                new Query.FilterPredicate(
-                        "userId", Query.FilterOperator.EQUAL, newMessage.recipientId));
-        Entity userEntity = datastore.prepare(userQuery).asSingleEntity();
-        User recipient = new User(userEntity);
-
-        // Translate the text.
-        Translate translate = TranslateOptions.getDefaultInstance().getService();
-        String lang = recipient.language;
-        Translation translation =
-                translate.translate(
-                        newMessage.text, Translate.TranslateOption.targetLanguage(lang));
-        String translatedText = translation.getTranslatedText();
-
-        // Add fields to message and add a new entity.
-        newMessage.messageId = UUID.randomUUID().toString();
-        newMessage.chatroomId = chatroomID;
-        newMessage.translatedText = translatedText;
-        newMessage.timestamp = System.currentTimeMillis();
-        newMessage.setEntity();
-
-        // Push the message.
-        PusherAPI pusherStore = new PusherAPI();
-        Pusher pusher =
-                new Pusher(pusherStore.getID(), pusherStore.getKey(), pusherStore.getSecret());
-        pusher.setCluster("us2");
-        pusher.setEncrypted(true);
-
-        // New message instance.
-        Gson gson = new Gson();
-
-        pusher.trigger(newMessage.senderId, "new-message", gson.toJson(newMessage));
-        pusher.trigger(newMessage.recipientId, "new-message", gson.toJson(newMessage));
-    }
-}
```

Comment:
Why are we deleting this? Seems like application would crash without this

Reply:
?Wait... I did not intend to delete that. My bad I'm gonna fix that real quick

Reply:
That's weird. I didn't delete that. Thanks for catching that.